src/client/components/Text/SmallCenteredText.tsx

```diff
@@ -2,10 +2,10 @@ import styled from 'styled-components';
 
 interface Props {
 	color?: string;
-	margin?: string;
-	fontSize?: string;
 	fontFamily?: string;
+	fontSize?: string;
 	fontWeight?: string;
+	margin?: string;
```

Comment:
Yay for alphabetizing fields

Reply:
Interfaces can't be fixed automatically, but it should be simpler to keep them that way now.